extension.ts

```diff
@@ -67,6 +67,10 @@ const findVSCode = (): CodeAppInfo | null => {
     // Offical VSCode snap, see <https://github.com/Jomik/vscode-search-provider/pull/14>
     // and <https://snapcraft.io/code>
     ["code_code.desktop", "Code"],
+    // VSCodium support, Free/Libre Open Source Software Binaries of VSCode. 
```

Comment:
You had a trailing whitespace on this line, hence the CI failure.  I merged this change and fixed the formatting issue.